megamek/src/megamek/common/FixedWingSupport.java

```diff
@@ -417,6 +417,7 @@ public double getPriceMultiplier() {
             case STATION_KEEPING:
                 priceMultiplier = 1 + weight / 75.0;
                 break;
+            default:
```

Comment:
Better would be
```
default:
   break; 
```

Reply:
Yep!  My bad.  Don't want somebody accidentally adding a case statement below the default and have it fall through unintended.